app/src/main/java/org/mozilla/fenix/library/history/HistoryFragment.kt

```diff
@@ -193,6 +199,17 @@ class HistoryFragment : LibraryPageFragment<HistoryItem>(), UserInteractionHandl
         else -> super.onOptionsItemSelected(item)
     }
 
+    private fun getMultiSelectSnackBarMessage(historyItems: Set<HistoryItem>): String {
+        return if (historyItems.size > 1) {
+            getString(R.string.history_delete_multiple_items_snackbar)
+        } else {
+            getString(
+                R.string.history_delete_single_item_snackbar,
+                historyItems.first().url.toShortUrl(context!!.components.publicSuffixList)
```

Comment:
Instead of a double bang can we use `requireComponents()` here?

Reply:
Yes, modified it.